strapi/strapi
# Avoid content type `pluralName` DB clashes

### What does it do?

Detects `collectionName` clashes with Strapi admin DB tables when creating content types.

### Why is it needed?

Previously, creating a content type with a clashing name would cause Strapi to crash (see issue)

### How to test it?

Attempt to create a content type with the `pluralName`=`admin-roles`

When Strapi restarts you should see an error like this in the terminal
<img width="995" alt="image" src="https://user-images.githubusercontent.com/48524071/193864444-522b3f96-ee0b-456f-be57-3bc7cf833dc6.png">

### Related issue(s)/PR(s)

Fixes #13893
